File: dongyimai_sellergoods_interface/src/main/java/com/offcn/sellergoods/service/BrandService.java
package com.offcn.sellergoods.service;

import com.offcn.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    //public PageResult findPage(int page ,int rows);

    public void add(TbBrand brand);

    public void update(TbBrand brand);

    public TbBrand findOne(Long id);

    public void delete(Long[] ids);

    public PageResult findPage(TbBrand brand,int page ,int rows);

    List<Map> selectOptionList();

    public PageResult findPage(int page ,int rows);
}

File: dongyimai_manager_web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    this.save=function (entity) {
        var method="add";
        if(entity.id!=null){
            method="update";
        }
        return $http.post("../brand/"+method+".do",entity);
    }
    this.findOne=function (id) {
        return $http.get("../brand/findOne.do?id="+id);
    }

    this.dele=function (selectIds) {
        $http.get("../brand/delete.do?ids=" + selectIds);
    }

    this.findBrandList=function () {
        return $http.get('../brand/selectOptionList.do');
    }

    this.findPage=function (page,rows) {
        return $http.post("../brand/findPage.do?page="+page+"&rows="+rows);
    }
    this.search=function (page,rows,searchEntity) {
        return $http.post("../brand/search.do?page="+page+"&rows="+rows,searchEntity);
    }
})